### ISSUE TITLE
Doesn't run in react native

### ISSUE THREAD

**Issue by jlkalberer** [adamhalasz/uniqid#24]
I guess the `os` package isn't supported in react native.

I see this error:
```
bundling failed: Error: Unable to resolve module `os` from `C:\dev\node_modules\uniqid\index.js`: Module `os` does not exist in the Haste module map
```

**Comment by Juliocbr** [comment]
Same error in react native

**Comment by guvenkaranfil** [comment]
Did you resolve this issue?

**Comment by y7usuf** [comment]
I'm getting the same error in react native 0.59.9. Any update for this issue?

**Comment by elisechant** [comment]
Yes, it requires operating system network interfaces -> https://github.com/adamhalasz/uniqid/blob/master/index.js#L17

Action: update the documentation to say, this package does not support React Native development.

**Comment by akanshSirohi** [comment]
@adamhalasz Make it available for react native also, it will be very helpful!
